index.html

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta charset="ISO-8859-1">
+	<title>ROCA Observation Tool</title>
+	<link rel="stylesheet" href="obs.css">
+	<link rel="stylesheet" href="popupCSS.css">
+	<link rel="stylesheet" href="themify-icons.css">
+	<link rel="stylesheet" href="fonts/stylesheet.css">
+	<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+	<script type="text/javascript" src="obs.js"></script>
+	<script type="text/javascript">
+	function GetURLParameter(sParam)
+	{
+   		var sPageURL = window.location.search.substring(1);
+    	var sURLVariables = sPageURL.split('&');
```

Comment:
Were you thinking of having one parameter for building name and one for room number?